go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/RedisTimeSeries/redistimeseries-go
+
+go 1.14
+
+require (
+	github.com/gomodule/redigo v1.8.2
```

Comment:
@filipecosta90 did we downgrade it on purpose?

Reply:
@danni-m I believe we should keep the versions at the minimum compatible version as possible. 
Also, I've noticed that I set go version to 1.14 to 1.12 ( same as other modules ).